set.mm

```diff
@@ -525774,6 +526000,155 @@ elements in a zero ring (see ~ 0ringirng ).  (Contributed by Thierry
+        DNCVYDWVAUDVYJUSUWKWAYBVGYMYPAUWNWLYRUWLAWVEVVOWUTVGVLWVFVVPUYPDNPVGVVM
+        SWVAUDUOVVNURUWOWAZUWPUWQWVHYSYHZUWRWUKWUJYAVUMUYPDNVUQMVUIVUNUDVVDWUOW
+        VAVUIVIVXGUWSUWTAVUKRMJUXKVFZVFZVFVWGVGRKVDZGVDZXLVUOVFZRGVDZYBVUOVFZEV
+        DZEVDZTARVUJWVKAVUJMWVJVUMUXBZVFWVKAMVUEWVSAVUMVUELJNWVJQDVUEVIZUEUDUAV
+        VDWVJVIZVVTVWBUXAVMAMVUMWVJVWFUXCYSVMAVWGWVNWVPJYTVFZEJGWWBVNVFZVUAKVUO
+        QMWVJRWWBVIZWWAUEWWCVIZUFUGUHVUOVIVUBVWGVIWVNVIWVPVIVVTANUXDVFZVNVFZWWC
+        MAMVUMWWGWWCUXLVWFAVUMWWGJWWBLDNWWCWWFWWDUAUDVVDVWBWWFVIWWGVIWWEUXEXIUX
+        FAMVUAVFUYQVGAVUMVUAMJLDNUAVUBUDVVDVWFVWBUXGWVIYSUKUXHAWVRWVLBRGVDZCEVD
+        ZEVDTAWVMWVLWVQWWIEAWVMVWHWVLGVDWVLAVWGVWHWVLGWUKUXIAQJGVWHWVLUEUFWUHWU
+        GAQKJWFVFZVGRQJWWJWWJVIZUEWGUHAWUEWWJWRVJWUGJWWJWWKWJWIVVPUKWMUXJYSAWVO
+        WWHWVPCEAWVNBRGAWVNXLVWIVFZBXLVUOVWIVWKWSAWWLXLVWLVFZXLVWNVFZVWPVDZVWRB
+        VWPVDBAVVEVWTVXAXLWKVJZWWLWWOVLVVJVVSVWEWWPAUXMWLZVUMVWPFNVURVVBXLDUDVV
+        DUPVXBWQWTAWWMVWRWWNBVWPAVBXLVXEVWRWKVWLXBVXHAVYLXCZVXDVUQVWRWWRVXDXMZX
+        LVGVLZXMZXLVGXTYCVYLWWSWXAUYLAVYLVXDWWTVXCXLVGXOUXNXQUXOXRWWQVYPXGAWWNX
+        LVXLVFZXLVXNVFZVWPVDZBVWRVWPVDBAVVEVXQVXRWWPWWNWXDVLVVJVWCVWDWWQVUMVWPF
+        NVUTVVAXLDUDVVDUPVXBWQWTAWXBBWXCVWRVWPAWXBBXLVXSVFZVYAVDZBVUQVYAVDBAVVE
+        VYEVVQWWPWXBWXFVLVVJVYIVVRWWQIVUMDNHVYAVYDBSXLUDVVDVYJUSUQVYKXJXKAWXEVU
+        QBVYAAVBXLVYMVUQWKVXSXBVYNWWRVYLVUQVWRAVYLXDXEWWQVXKXGXSAVYDNVYAVUQBVYJ
+        VYKVXGVVJVYIUXPYAAVBXLVYSVWRWKVXNXBWUBWWRVYRCVWRVYLVYRXMAVYLVYRXLYBVLYB
+        XLUXQXNVXCXLYBXOXPXQXRWWQVYPXGYDAVYDVWPNBVWRVYJVXBVXFWUDVYIYEYAYDAVYDVW
+        PNBVWRVYJVXBVXFWUDVYIUXRYAYHUXIAWVPYBVWIVFZCYBVUOVWIVWKWSAWXGYBVWLVFZYB
+        VWNVFZVWPVDZVWRCVWPVDZCAVVEVWTVXAYBWKVJZWXGWXJVLVVJVVSVWEWXLAUXSWLZVUMV
+        WPFNVURVVBYBDUDVVDUPVXBWQWTAWXHVWRWXICVWPAVBYBVXEVWRWKVWLXBVXHAVYRXCZVX
+        DVUQVWRVYRWWSAWUCXQXRWXMVYPXGAWXIYBVXLVFZYBVXNVFZVWPVDZWXKCAVVEVXQVXRWX
+        LWXIWXQVLVVJVWCVWDWXMVUMVWPFNVUTVVAYBDUDVVDUPVXBWQWTAWXOVWRWXPCVWPAWXOB
+        YBVXSVFZVYAVDZVYCVWRAVVEVYEVVQWXLWXOWXSVLVVJVYIVVRWXMIVUMDNHVYAVYDBSYBU
+        DVVDVYJUSUQVYKXJXKAWXRVWRBVYAAVBYBVYMVWRWKVXSXBVYNWXNVYLVUQVWRVYRVYOAVY
+        RVYLYBXLVLYBXLUXQYCVXCYBXLXOXPXQXRWXMVYPXGXSVYQYAACWXPAVVEVYTCWXPVLVVJW
+        UAIDNVYDCUDUSVYJUXTWAUYAYDAVYDVWPNCVWRVYJVXBVXFWUDWUAUXRZYAYDWXTYAYHYDY
+        DUNYSYAZUYBAQJLNVAVUERTWVTUAUEUCVVTVWBUYCUYDUYEAVUMVUAUYSJLDNUAVUBUDVVD
+        ARQDRVCVKVUEVFVFTVLVCVUEUYFUYGZJLVVGUYHVFZDUYIVFZUYRVUETVCWVTDNYTVFVVGY
+        TVFUDNVVGYTUAWPUYJZUEUIUJUKUCWYBVIWYDVIWYCVIVUCUYKVWBUXGYSARQUYPDVUMJLM
+        UYRVUETVUNWVTWYEUEUIUJUKUCVUCNVVGVEUAWPWUOVVDWYAVWFWURUYMUYNWVIYQ $.
+    $}
+
+    $( If an element ` X ` is a solution of a quadratic equation, then it is
+       either in the base field, or in a quadratic field extension.
+       (Contributed by Thierry Arnoux, 22-Jun-2025.) $)
+    fldsqrextdeg2 $p |- ( ph -> ( X e. F \/ ( L [:] K ) = 2 ) ) $=
```

Comment:
typo? (df-extdg vs fldsqrextdeg2)

I guess fldsqr refers to a quadratic field? Though, extdeg2 already kinda says that. Very minor nitpick

maybe rtextdg2 would be clearer? optional